src/VisualStudio/Core/Impl/Options/AbstractCodeStyleOptionViewModel.cs

```diff
@@ -32,12 +33,16 @@ internal abstract class AbstractCodeStyleOptionViewModel : AbstractNotifyPropert
         public double DescriptionMargin { get; set; }
         public string GroupName { get; set; }
         public abstract CodeStylePreference SelectedPreference { get; set; }
-        public abstract NotificationOptionViewModel SelectedNotificationPreference { get; set; }
         public List<CodeStylePreference> Preferences { get; set; }
         public List<NotificationOptionViewModel> NotificationPreferences { get; set; }
         public abstract bool NotificationsAvailable { get; }
 
         public virtual string GetPreview() => SelectedPreference.IsChecked ? _truePreview : _falsePreview;
+        public virtual NotificationOptionViewModel SelectedNotificationPreference
+        {
+            get { return NotificationPreferences.First(); }
+            set { }
```

Comment:
Ahh, once all the code style options support these settings, then you can make it abstract instead of just virtual, right?